### PR TITLE
 Google Domain - Dynamic DNS + Wildcards

### PR DESCRIPTION
**Pull request by gauravspatel**
I had to struggle through reverse engineering the /nic/update call and figure out why I kept getting nohost. I discovered that I need to setup Dynamic DNS and also figured out how to do wildcards. Adding to the README so others can save time.

Great image! Thanks for the work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,6 +217,8 @@ DYNDNS:
 
 Google:
 
+Note that you must enable Dynamic DNS in the Synthetic records section of DNS management. The username and password is generated once you . You cannot create a wildcard entry for Dynamic DNS but you can CNAME `"*"` to the `"@"` domain for the same effect.
```
Comment (qdm12) [inline README.md:220]: There is a missing piece after `is generated once you`.

Comment (qdm12) [inline README.md:220]: > You cannot create a wildcard entry for Dynamic DNS but you can CNAME `"*"` to the `"@"` domain for the same effect.

You mean that setting a CNAME record with key `"*"` and value `"@"` has the same effect right?

**Review by qdm12 — COMMENTED** [review]
Thanks for the PR! Can you please just comment back on my comments to clarify a few things. I'll add it in the [Google Wiki section](https://github.com/qdm12/ddns-updater/wiki/Google) to keep the readme cleaner.

File: README.md (hunk 1 above)
Comment (gauravspatel) [reply README.md:220]: The text should read:

Note that you must enable Dynamic DNS in the Synthetic records section of DNS management. The username and password is generated once you create the Dynamic DNS entry . You cannot create a wildcard entry for Dynamic DNS but you can CNAME `"*"` to the `"@"` domain for the same effect.

Comment (gauravspatel) [reply README.md:220]: Great catch...Yes, same effect.  I updated my text in the previous comment to help clarify.

**Comment by qdm12** [comment]
I'll add your contribution in #128, closing this one for now. I mentioned you in the part you added thanks for the contribution!

**Pull request closed without merging**